src/operations/hulls/hullVectors2.js

```diff
@@ -20,15 +20,15 @@ const compareIndex = (index1, index2) => {
 }
 
 // Ported from https://github.com/bkiers/GrahamScan
-const compute = (points) => {
```

Comment:
@z3dev please do not hate me for this, but I think the 'points' naming was clearer, very sorry, The only thing I was not sure of is the name of the function itself, I can honestly live with the previous naming, It was me mostly thinking out lout about coherence in naming our functions, sorry !

Reply:
:) No problem. Actually, I think we should stick with a common convention anyway, which puts the 2/3 at the end. So, the correct names would be hullPoints2 / hullPoints3